Mudança para melhor experiência do usário

nicolexerxas opened the pull request:
Adicionei em primeiro momento o botão autenticar, para seguir a lógica do programa, em um segundo momentyo, adicionei o botão limpar, caso haja algum erro pelo usuário. O que vocês acham que posso fazer para melhorar a experiência do usuário?

nicolexerxas committed:
Adição de botão "Autenticar"

Linha 82 foi adicionado o botão Autenticar para melhor experiência do usuário

nicolexerxas committed:
Adição de botão "Limpar"

Adição de botão "Limpar" para melhor experiência do usuário

jhbergallo commented:
Achei muito interessante tua ideia de adicionar funcionalidades com o intuito de melhorar a experiência do usuário! Acho que uma coisa legal que tu pode adicionar são comentários ao longo do código pra ele ficar mais "legível". Em geral tá muito bom!!

nicolexerxas committed:
Comentarios

Comentarios foram adicionados para melhgor entendimento do código, como sugerido
